nf_core/modules/lint.py

```diff
@@ -141,8 +141,12 @@ def lint_local_modules(self, local_modules):
                 mod_object.main_nf = mod
                 mod_object.module_name = os.path.basename(mod)
                 mod_object.lint_main_nf()
-                self.warned += mod_object.warned + mod_object.failed
-                self.passed += mod_object.passed
+                warned = []
+                passed = []
+                warned += mod_object.warned + mod_object.failed
+                passed += mod_object.passed
+                self.passed = [(mod_object, m) for m in passed]
+                self.warned = [(mod_object, m) for m in warned]
```

Comment:
This should be equivalent I think?

```suggestion
                self.passed = [(mod_object, m) for m in mod_object.passed]
                self.warned = [(mod_object, m) for m in mod_object.warned + mod_object.failed]
```